### PR TITLE
[Polygon Edge]Add extra env parameters to Blockscout

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Comment by DannyS03** [comment]
@laviniat1996 feel free to use the Edge label for PRs regarding Edge content

File: docs/edge/additional-features/blockscout.md [hunk 1]
```diff
@@ -275,6 +275,9 @@ ETHEREUM_JSONRPC_TRACE_URL=https://rpc.poa.psdk.io:8545
 DATABASE_URL=postgresql://blockscout:Passw0Rd@db.instance.local:5432/blockscout
 SECRET_KEY_BASE=VTIB3uHDNbvrY0+60ZWgUoUBKDn9ppLR8MI4CpRz4/qLyEFs54ktJfaNT6Z221No
 HEART_COMMAND="systemctl start explorer.service"
+TRACE_FIRST_BLOCK=false
+TRACE_LAST_BLOCK=false
+INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER=true
```
Comment (zivkovicmilos) [inline docs/edge/additional-features/blockscout.md:280]: @laviniat1996 @ZeljkoBenovic 

We should probably note that some of these are changeable, while others are not

**Review by zivkovicmilos — CHANGES_REQUESTED** [review]
Looks okay, left a minor suggestion

**Comment by laviniat1996** [comment]
> @laviniat1996 feel free to use the Edge label for PRs regarding Edge content

@DannyS03  Thank you for the tip 👍

File: docs/edge/additional-features/blockscout.md (hunk 1 above)
Comment (cerberushades) [reply docs/edge/additional-features/blockscout.md:280]: ok guys, should we merge it @laviniat1996 @zivkovicmilos

**Review by cerberushades — APPROVED** [review]

**Comment by zivkovicmilos** [comment]
Hey @cerberushades,

I believe you gave us contributor access to the repo, meaning we can squash and merge PRs.

We usually have thorough internal discussions on open documentation PRs, and don't merge anything until everything is resolved and everyone is satisfied 🙂

You can leave the squash and merge of Edge PRs to the Edge team, we won't leave them hanging as they're accounted for in our internal task tracking system 🍻 

I wouldn't want for these PRs to get merged by someone from outside of the Edge team.

File: docs/edge/additional-features/blockscout.md (hunk 1 above)
Comment (ZeljkoBenovic) [reply docs/edge/additional-features/blockscout.md:280]: @laviniat1996 Maybe just add a comment above these three env vars like:
`# these environment variables disable debug tracing as it is not currently supported in Polygon Edge`

**Review by ZeljkoBenovic — APPROVED** [review]

File: docs/edge/additional-features/blockscout.md (hunk 1 above)
Comment (laviniat1996) [reply docs/edge/additional-features/blockscout.md]: @ZeljkoBenovic Done 24bb49b18da67709d975b5cb5bc4dd00dddb84ad

**Review by zivkovicmilos — APPROVED** [review]
Looks good now, thank you for clarifying the env file 🙏

**Pull request merged**